functions-and-scope.js

```diff
@@ -16,20 +16,41 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+let output = 0;
+
+for (let i = 0; i < grades.length; i++) {
+    if ( grades[i] >= 8 )   {
+        output += 1;
```

Comment:
Je kan dit nog iets korter schrijven:
```suggestion
        output++;
```